src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/HiddenFieldCheck.java

```diff
@@ -125,6 +125,28 @@
  * <pre>
  *  &lt;module name=&quot;HiddenField&quot;/&gt;
  * </pre>
+ * <pre>
+ * public class SomeClass {
+ *
+ *   private String field; // field 1
+ *   private String test; // field 2
```

Comment:
please remove trailing comments.
you can name field like:
```
 *   private String field; 
 *   private String testField; 
```

to clearly see in code below is it a field of not.

Reply:
done